metpy/calc/thermo.py

```diff
@@ -1038,3 +1038,99 @@ def most_unstable_parcel(p, temperature, dewpoint, heights=None,
     theta_e = equivalent_potential_temperature(p_layer, T_layer)
     max_idx = np.argmax(theta_e)
     return p_layer[max_idx], T_layer[max_idx], Td_layer[max_idx]
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def mixed_parcel(p, temperature, dewpoint, parcel_start_pressure=None, bottom=None,
```

Comment:
Order of `bottom`, `heights`, `depth` should be `heights`, `bottom`, `depth` to match everywhere else. This is already the order in the docstring below.